src/dmd/globals.h

```diff
@@ -172,6 +172,7 @@ struct Param
 
     CHECKACTION checkAction;       // action to take when bounds, asserts or switch defaults are violated
 
+    unsigned recursionLimit;
```

Comment:
I think you have to remove the old enum too? I might be thinking of the cxx branch though.

Reply:
In the last commit I removed the enum from frontend.h and replaced it, based on the Param::errorLimit variable that has a similar behavior